### DISCUSSION TITLE
自己训练的检测模型和识别模型，分别使用精度都很高，串联起来之后识别精度大幅降低

### DISCUSSION THREAD

**Discussion by UstbLjc** [PaddlePaddle/PaddleOCR#12540] [Q&A]
<img width="641" alt="1" src="https://github.com/PaddlePaddle/PaddleOCR/assets/35488143/7edfc8db-9101-4400-8c46-554fb50c3dd7">
检测模型的结果图像直接传递给识别模型无法正确识别，本地保存为jpg图片读取后传递给识别模型，结果正常。使用的PaddleX导出的FD部署包。

**Comment by M1aoLiu** [comment]
我也遇到这个问题了，请问您解决了吗

**Comment by M1aoLiu** [comment]
> 我也遇到这个问题了，请问您解决了吗

我找到了解决方案：https://github.com/PaddlePaddle/PaddleOCR/issues/3822
